source/renderer/app/api/staking/requests/submitRedeemItnRewards.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+// @REDEEM TODO: Remove when the API endpoint is implemented
+/* eslint-disable */
+import type { RequestConfig } from '../../common/types';
+import type {
+  SubmitRedeemItnRewardsRequest,
+  SubmitRedeemItnRewardsResponse,
+} from '../types';
+
+const rewardsTotal = 1000;
+const transactionFees = 1000;
+const finalTotal = 1000;
```

Comment:
@daniloprates maybe would be better to isolate this to some config file as CONST

Reply:
This is temporary. Because we don't have the required endpoint.

Reply:
@darko-mijic ok, thanks for explanation